spec/swagger.yaml

```diff
@@ -64,7 +64,7 @@ tags:
       Retrieve summary information about your customers, subscriptions, transactions, and more.
   - name: Exports
     description: |
-      Bulk data exports
+      Bulk data exports. You can schedule bulk data exports on a recurring basis if needed.
```

Comment:
Any chance we can remove the work bulk from this description and just use `data exports`?